plot_cfDNA_scATAC/scATAC_demo.Rmd

```diff
@@ -0,0 +1,430 @@
+## Make UMAP
+
+```{r umap, fig.align='center', echo=TRUE, warning=FALSE, message=F}
+
+pbmc <- RunUMAP(object = pbmc, reduction = 'lsi', dims = 2:30)
+pbmc <- FindNeighbors(object = pbmc, reduction = 'lsi', dims = 2:30)
+pbmc <- FindClusters(object = pbmc, verbose = FALSE, algorithm = 3)
+DimPlot(object = pbmc, label = TRUE) + NoLegend()
+```
+
+## Create a gene activity matrix
+
+```{r gene_activity, fig.align='center', echo=TRUE, warning=FALSE, message=F}
+
+# get the activities of the genes from the peaks
+gene.activities <- GeneActivity(pbmc)
+
+# add the gene activity matrix to the Seurat object as a new assay and normalize it
+pbmc[['RNA']] <- CreateAssayObject(counts = gene.activities)
+pbmc <- NormalizeData(
+  object = pbmc,
+  assay = 'RNA',
+  normalization.method = 'LogNormalize',
+  scale.factor = median(pbmc$nCount_RNA)
+)
+
+DefaultAssay(pbmc) <- 'RNA'
+
+DimPlot(object = pbmc, reduction = "umap")
+
+FeaturePlot(
+  object = pbmc,
+  features = c('MS4A1', 'CD3D', 'LEF1', 'NKG7', 'TREM1', 'LYZ'),
+  pt.size = 0.1,
+  max.cutoff = 'q95',
+  ncol = 3
+)
+
+```
+
+
+## now get labels from scRNASeq
+
+
+```{r get_rnaseq, fig.align='center', echo=TRUE, warning=FALSE, message=F}
+
+# Load the pre-processed scRNA-seq data for PBMCs
+expr_file = paste0(proj_dir, "/data/pbmc_10k_v3.rds")
+
+pbmc_rna <- readRDS(expr_file)
+
+transfer.anchors <- FindTransferAnchors(
+  reference = pbmc_rna,
+  query = pbmc,
+  reduction = 'cca'
+)
+
+predicted.labels <- TransferData(
+  anchorset = transfer.anchors,
+  refdata = pbmc_rna$celltype,
+  weight.reduction = pbmc[['lsi']],
+  dims = 2:30
+)
+
+pbmc <- AddMetaData(object = pbmc, metadata = predicted.labels)
+
+```
+
+```{r plot_rnaseq, fig.align='center', echo=TRUE, warning=FALSE, message=F}
+
+plot1 <- DimPlot(
+  object = pbmc_rna,
+  group.by = 'celltype',
+  label = TRUE,
+  repel = TRUE) + NoLegend() + ggtitle('scRNA-seq')
+
+plot2 <- DimPlot(
+  object = pbmc,
+  group.by = 'predicted.id',
+  label = TRUE,
+  repel = TRUE) + NoLegend() + ggtitle('scATAC-seq')
+
+plot1 + plot2
+
+```
+
+now match clusters from RNA-Seq to ATAC-Seq
+
+```{r match_rnaseq, fig.align='center', echo=TRUE, warning=FALSE, message=F}
+
+#pbmc <- subset(pbmc, idents = 14, invert = TRUE)
+pbmc <- RenameIdents(
+  object = pbmc,
+  '0' = 'CD14 Mono',
+  '1' = 'CD4 Memory',
+  '2' = 'CD8 Effector',
+  '3' = 'CD4 Naive',
+  '4' = 'CD14 Mono',
+  '5' = 'DN T',
+  '6' = 'CD8 Naive',
+  '7' = 'NK CD56Dim',
+  '8' = 'pre-B',
+  '9' = 'CD16 Mono',
+  '10' = 'pro-B',
+  '11' = 'DC',
+  '12' = 'NK CD56bright',
+  '13' = 'pDC'
+)
+pbmc <- RenameIdents(
+  object = pbmc,
+  'CD14 Mono' = 'Mono',
+  'CD4 Memory' = 'T-cells',
+  'CD8 Effector' = 'T-cells',
+  'CD4 Naive' = 'T-cells',
+  'CD14 Mono' = 'Mono',
+  'DN T' = 'T-cells',
+  'CD8 Naive' = 'T-cells',
+  'NK CD56Dim' = 'NK',
+  'pre-B' = 'B-cells',
+  'CD16 Mono' = 'Mono',
+  'pro-B' = 'pro-B',
+  'DC' = 'DC',
+  'NK CD56bright' = 'NK',
+  'pDC' = 'DC'
+)
+pbmc <- RenameIdents(
```

Comment:
I'm confused what you're doing here. Are you compressing 'CD4 Naive' to 'T-cells' and then to 'Other'?

Reply:
eek yeah sorry, this is sloppy. I was going back and forth trying to figure out which labeling was better. Clarified here: https://github.com/greenelab/cfDNA_atac_demo/pull/3/commits/e1b7e639f576324543ec6a2d47756582a56f2758